src/orangeClockFunctions/displayBlockMoscowFees.py

```diff
@@ -185,10 +193,10 @@ def main():
                 textRow2 = getMoscowTime()
             elif dispVersion2 == "fp1":
                 symbolRow2 = "K"
-                textRow2 = str(getPrice("USD"))
+                textRow2 = str(getFiatPrice("USD"))

```

Comment:
Don't need the cast to string (`str()`) now that the function is returning a string.